Nodejs/Setup/NodejsTools/NodejsTools.wxs

```diff
@@ -15,6 +15,7 @@
             <Directory Id="Dir_$(var.key)TestFrameworks" Name="TestFrameworks">
                 <Directory Id="Dir_$(var.key)TestFrameworksExportRunner" Name="ExportRunner" />
                 <Directory Id="Dir_$(var.key)TestFrameworksMocha" Name="Mocha" />
+                <Directory Id="Dir_$(var.key)TestFrameworksTape" Name="Tape" />
```

Comment:
I've been able to get the installer built. I'm a PowerShell newb, but you have good documentation!